distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -538,14 +538,17 @@ type Date
             _ ->
                 Error.throw (Illegal_Argument.Error "Illegal period argument")
 
-    ## A Date to Json conversion.
+    ## Convert to a JavaScript Object representing this Date.
```

Comment:
This comment sounds like the result of this operation would be a JS `Date` - which is not the case. Can we rephrase this to be unambiguous that it is ...?

Hmm what is it really - an Enso-oriented JS Object representation? Not sure how to call this.